Main.java

```diff
@@ -9,7 +11,25 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a Scanner for user input 
+    Scanner input = new Scanner(System.in);  
+
+    // prompt the user to enter an integer 
+    System.out.println("Please enter a integer"); 
+    int A = input.nextInt(); 
+
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for. So this should start with a lowercase letter and be more descriptive.